code/game/machinery/vending/marine_vending.dm

```diff
@@ -129,6 +129,7 @@
 					/obj/item/storage/box/nade_box/M15 = 1,																																																																										
 					/obj/item/weapon/gun/smg/ppsh = 4,
 					/obj/item/weapon/gun/shotgun/double = 2,
+					/obj/item/weapon/gun/shotgun/pump/bolt = 2,
```

Comment:
this list does nothing in a normal round, its overriden in gamemode

Reply:
but it does show up in a normal round. cause people have been finding it in the contraband sections of the appropriate vendor.